Header header
float64[] confidence 
